### PR TITLE
Tests for benchmarks and baselins

### PR DESCRIPTION
**Pull request by kjang96**
Added tests for running all benchmarks and rename test_benchmarks.py to test_baselines.py, which is what it actually is

### CODE REVIEW

File: tests/slow_tests/test_benchmarks.py [hunk 1]
```diff
@@ -1,37 +1,94 @@
...
+
+        # Register as rllib env
+        register_env(env_name, create_env)
+
+        alg = ppo.PPOAgent(
+            env=env_name, config=config)
+
+        for i in range(num_runs):
+            alg.train()
+            checkpoint_path = alg.save()
```
Comment (eugenevinitsky) [inline tests/slow_tests/test_benchmarks.py]: Can you remove the checkpoint files afterwards?

**Comment by eugenevinitsky** [comment]
The ray failure is due to the travis yml not being up to date with what is on master; copy over the parameters to get it to pass.

**Comment by eugenevinitsky** [comment]
Most importantly the line pip install redist ~=0.26.1

File: tests/slow_tests/test_benchmarks.py (hunk 1 above)
Comment (kjang96) [reply tests/slow_tests/test_benchmarks.py]: Addressed!

File: tests/slow_tests/test_benchmarks.py [hunk 2]
```diff
@@ -1,37 +1,101 @@
...
+        shutil.rmtree('benchmark_tmp')
+
+    def ray_runner(self, num_runs, flow_params, version):
+        alg_run = 'PPO'
+        HORIZON = 10
+
+        agent_cls = get_agent_class(alg_run)
+        config = agent_cls._default_config.copy()
+        config['num_workers'] = 1
+        config['train_batch_size'] = 200  # arbitrary
```
Comment (eugenevinitsky) [inline tests/slow_tests/test_benchmarks.py]: can you try decreasing train_batch_size to 50 or smaller and see if that gives a little speed_boost?

**Comment by eugenevinitsky** [comment]
@kjang96 added two minor comments and then will merge it

**Comment by kjang96** [comment]
^ whoops ignore latest commit

**Pull request merged**